### PR TITLE
Use "instance" consistently

### LINKED ISSUE #1147: Server or instance?
#### Context

This Issue follows a [previous discussion] and is added for visibility.

#### Goal

It aims to clarify some terms and their use.

#### Question

A server can host multiple instances. Are there any settings allowing to block a whole server or is the management only per instance?

#### Proposal

- If a whole server can be blocked, we could be specific when talking about and instance or server.
- If only servers can be blocked, we could change all occurrences of _instance_ to _server_ in file names, within the files and the locales.
- If the current _server_ occurrences only refer to bookwyrm instances (not the place hosting the instance), we could change all occurrences of _server_ to _instance_ in file names, within the files and the locales.

[previous discussion]: https://github.com/bookwyrm-social/bookwyrm/pull/935/files#r624529000

### PR DESCRIPTION
**Pull request by mouse-reeve**
changes the wording in the app to call instances "instances" instead of servers. Fixes #1147

### CODE REVIEW

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1224,77 +1223,85 @@ msgstr "Chercher un livre ou un compte"
...
 msgid "Join"
 msgstr "Rejoindre"
 
-#: bookwyrm/templates/layout.html:211
-msgid "About this server"
+#: bookwyrm/templates/layout.html:206
+#, fuzzy
+#| msgid "About this server"
+msgid "About this instance"
 msgstr "À propos de ce serveur"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1289]:
```suggestion
#: bookwyrm/templates/layout.html:206
msgid "About this instance"
msgstr "À propos de cette instance"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1522,8 +1530,9 @@ msgid "Resolve"
 msgstr "Résoudre"
 
 #: bookwyrm/templates/moderation/reports.html:6
-#, python-format
-msgid "Reports: %(server_name)s"
+#, fuzzy, python-format
+#| msgid "Reports: %(server_name)s"
+msgid "Reports: %(instance_name)s"
 msgstr "Signalements : %(server_name)s"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1536]:
```suggestion
msgid "Reports: %(instance_name)s"
msgstr "Signalements : %(instance_name)s"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1533,9 +1542,10 @@ msgid "Reports"
 msgstr "Signalements"
 
 #: bookwyrm/templates/moderation/reports.html:14
-#, python-format
-msgid "Reports: <small>%(server_name)s</small>"
-msgstr "Signalements: <small>%(server_name)s</small>"
+#, fuzzy, python-format
+#| msgid "Users: <small>%(server_name)s</small>"
+msgid "Reports: <small>%(instance_name)s</small>"
+msgstr "Comptes : <small>%(server_name)s</small>"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1548]:
```suggestion
msgid "Reports: <small>%(instance_name)s</small>"
msgstr "Signalements : <small>%(instance_name)s</small>"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -1803,7 +1813,9 @@ msgstr "Gérer les comptes"
 #: bookwyrm/templates/settings/admin_layout.html:39
 #: bookwyrm/templates/settings/federation.html:3
 #: bookwyrm/templates/settings/federation.html:5
-msgid "Federated Servers"
+#, fuzzy
+#| msgid "Federated Servers"
+msgid "Federated Instances"
 msgstr "Serveurs fédérés"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1819]:
```suggestion
msgid "Federated Instances"
msgstr "Instances fédérées"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -1931,12 +1943,16 @@ msgstr "inactive"
 #: bookwyrm/templates/settings/federation.html:10
 #: bookwyrm/templates/settings/server_blocklist.html:3
 #: bookwyrm/templates/settings/server_blocklist.html:20
-msgid "Add server"
-msgstr "Ajouter un serveur"
+#, fuzzy
+#| msgid "View instance"
+msgid "Add instance"
+msgstr "Voir l’instance"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1949]:
```suggestion
msgid "Add instance"
msgstr "Ajouter une instance"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -1931,12 +1943,16 @@ msgstr "inactive"
...
 
 #: bookwyrm/templates/settings/edit_server.html:7
 #: bookwyrm/templates/settings/server_blocklist.html:7
-msgid "Back to server list"
+#, fuzzy
+#| msgid "Back to server list"
+msgid "Back to instance list"
 msgstr "Retour à la liste des serveurs"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:1956]:
```suggestion
msgid "Back to instance list"
msgstr "Retour à la liste des instances"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -2043,8 +2059,10 @@ msgstr "Tous les comptes de cette instance seront réactivés."
 
 #: bookwyrm/templates/settings/federation.html:19
 #: bookwyrm/templates/user_admin/server_filter.html:5
-msgid "Server name"
-msgstr "Nom du serveur"
+#, fuzzy
+#| msgid "Instance Name:"
+msgid "Instance name"
+msgstr "Nom de l’instance :"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:2065]:
```suggestion
msgid "Instance name"
msgstr "Nom de l’instance"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -2833,8 +2843,9 @@ msgid "Back to users"
 msgstr "Retour aux comptes"
 
 #: bookwyrm/templates/user_admin/user_admin.html:7
-#, python-format
-msgid "Users: <small>%(server_name)s</small>"
+#, fuzzy, python-format
+#| msgid "Users: <small>%(server_name)s</small>"
+msgid "Users: <small>%(instance_name)s</small>"
 msgstr "Comptes : <small>%(server_name)s</small>"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:2849]:
```suggestion
msgid "Users: <small>%(instance_name)s</small>"
msgstr "Comptes : <small>%(instance_name)s</small>"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -2851,8 +2862,10 @@ msgid "Last Active"
 msgstr "Dernière activité"
 
 #: bookwyrm/templates/user_admin/user_admin.html:38
-msgid "Remote server"
-msgstr "Serveur distant"
+#, fuzzy
+#| msgid "View instance"
+msgid "Remote instance"
+msgstr "Voir l’instance"
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:2868]:
```suggestion
msgid "Remote instance"
msgstr "Instance distante"
```

File: locale/fr_FR/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -2895,6 +2908,10 @@ msgstr "Rétablir le compte"
 msgid "Access level:"
 msgstr "Niveau d’accès :"
 
+#: bookwyrm/templates/widgets/clearable_file_input_with_warning.html:3
+msgid "File exceeds maximum size: 10MB"
+msgstr ""
```
Comment (joachimesque) [inline locale/fr_FR/LC_MESSAGES/django.po:2913]:
```suggestion
msgstr "Ce fichier dépasse la taille limite : 10 Mo"
```

**Review by joachimesque — COMMENTED** [review]
Here are some fixes for the French locale

File: locale/fr_FR/LC_MESSAGES/django.po (hunk 2 above)
Comment (mouse-reeve) [reply locale/fr_FR/LC_MESSAGES/django.po:1536]: oh I need to remember to fix this variable name on the other locales! thank you

**Pull request merged**